pkg/types/types.go

```diff
@@ -0,0 +1,31 @@
+package types
+
+import (
+	"time"
+)
+
+type Metadata struct{
+
+}
+type Spec struct{
+}
+
+type Status struct{
+}
+type ProviderCluster struct {
+	ClusterID   string
+	AccountID   string
+	SatelliteID string
+	MetaData    Metadata
+	Spec        Spec
+	Status      Status
+}
+
+type Request struct {
+	Name string `json:"name"`
+}
+
+type Result struct {
+	Requeue bool
+	After   time.Duration
+}
```

Comment:
Please Formen interface types back to the Formen package as requested in previous reviews